src/mParticle.Sdk/Model/DeviceInformation.cs

```diff
@@ -81,6 +81,51 @@ public enum PlatformEnum
         /// </summary>
         [DataMember(Name="platform", EmitDefaultValue=false)]
         public PlatformEnum? Platform { get; set; }
+
+        /// <summary>
+        /// Gets or Sets AttTimestampUnixtimeMs
+        /// </summary>
+        [DataMember(Name = "att_timestamp_unixtime_ms", EmitDefaultValue = false)]
+        public long? AttTimestampUnixtimeMs { get; set; }
+
+        /// <summary>
+        /// Defines AttAuthorizationStatus
+        /// </summary>
+        [JsonConverter(typeof(StringEnumConverter))]
+        public enum AttAuthorizationStatusEnum
+        {
+            /// <summary>
+            /// Enum authorized for value: authorized
+            /// </summary>
+            [EnumMember(Value = "authorized")]
+            Authorized = 1,
+
+            /// <summary>
+            /// Enum denied for value: denied
+            /// </summary>
+            [EnumMember(Value = "denied")]
+            Denied = 2,
+
+            /// <summary>
+            /// Enum notDetermined for value: notDetermined
+            /// </summary>
+            [EnumMember(Value = "not_determined")]
+            NotDetermined = 3,
+
+            /// <summary>
+            /// Enum restricted for value: restricted
+            /// </summary>
+            [EnumMember(Value = "restricted")]
+            Restricted = 4
+
+        }
+
+        /// <summary>
+        /// Gets or Sets AttAuthorizationStatus
+        /// </summary>
+        [DataMember(Name="attAuthorizationStatus", EmitDefaultValue=false)]
```

Comment:
this should have failed your tests, I think? should be snake_case?

Reply:
Ahh it caught it. But only as a warning.